test/sqllogictest/ldbc_bi_eager.slt

```diff
@@ -1449,24 +1446,17 @@ Explained Query:
                         Negate // { arity: 1 }
                           Project (#1) // { arity: 1 }
                             ReadIndex on=person_knows_person person_knows_person_person1id=[*** full scan ***] // { arity: 3 }
-                        Distinct project=[#0] // { arity: 1 }
-                          Union // { arity: 1 }
-                            Project (#0) // { arity: 1 }
-                              Get l7 // { arity: 2 }
-                            Constant // { arity: 1 }
```

Comment:
Something interesting happened here where it seems like we removed the `Constant` for some reason. Can't tell if it's a bug or a feature.